### PR TITLE
ResNet-50 || ImageNet Training || Setu Chokshi (MS AI MVP | PropertyGuru)

### PR DESCRIPTION
**Pull request by setuc**
ResNet-50 trained on 8 Tesla-V100 GPUs on Azure NDv2

I have used PyTorch 1.0 to train the models and only using it to train the model. I believe this is a significant update as this runs on a single machine without any heavy engineering to manage the clusters. Since it runs using basic PyTorch, anyone can run the models without an issue. 
I will upload the training code soon in a way that it is reproducible. Let me know it you have any questions. I am also planning to do a blog post on this so that people can train models beyond just resnet-50 with similar speeds that i have obtained.

### CODE REVIEW

**Comment by huan** [comment]
Great job @setuc !

File: ImageNet/train/setu_resnet50_azure_ndv2.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+	"version": "v1.0",
+	"author": "Setu Chokshi",
+	"authorEmail": "setuc@hotmail.com",
+	"framework": "PyTorch 1.0",
+	"model": "ResNet50",
+	"hardware": "Azure ND40s_v2",
+  "timestamp": "2019-04-13",
+	"costPerHour": 12.24,
```
Comment (codyaustun) [inline ImageNet/train/setu_resnet50_azure_ndv2.json]: Can you send me a link to the pricing information for an `ND40s_v2` instance from Azure?

Comment (setuc) [reply ImageNet/train/setu_resnet50_azure_ndv2.json]: https://azure.microsoft.com/en-us/pricing/details/virtual-machines/linux/

Please scroll down to NDv2 Series. You can find the pricing there. These are currently available East US region.

Comment (setuc) [reply ImageNet/train/setu_resnet50_azure_ndv2.json]: I fixed the filename, as per the required structure.

**Comment by codyaustun** [comment]
Looks good to me

**Comment by dfayzur** [comment]
Is the training code available?

Thanks for the work.

**Pull request merged**